src/i_video.c

```diff
@@ -315,16 +315,7 @@ static void AdjustWindowSize(void)
 {
     if (aspect_ratio_correct || integer_scaling)
     {
-        if (window_width * actualheight <= window_height * SCREENWIDTH)
```

Comment:
Please do not just kill Chocolate code. I'd prefer if it was commented out or if the check was extended with an impossible condition like `&& false` to the compiler will optimize it out. Also, whenever we change Chocolate code, we add a `// [crispy]` style comment.